ruby/rollbar.rb

```diff
@@ -0,0 +1,10 @@
+Rollbar.configure do |config|
+  config.access_token = ENV['ROLLBAR_ACCESS_TOKEN']
```

Comment:
Is het hier niet beter om wel `nil` te krijgen? Anders start de applicatie uberhaupt niet op. Volgens mij is dat erger dan een Rollbar config die het niet doet, dat kan je later wel fixen. Ik wil best `fetch` gebruiken, ik ben het er wel mee eens dat dat beter is, maar dan zou ik wel een fallback-waarde willen opgeven. Dus `nil`. 😄